.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--- Thanks for sending a Pull Request :heart: --->
+- Fixes #<Issue no.>
+
+<!--- Please read and understand everything below --->
+<!--- **Do not delete any text other than where you are instructed.** --->
+
+Check by changing each `[ ]` to `[x]` Please take note of the white-space as it matters.
```

Comment:
Remove this section from PR template, no one likes to tick things off after fixing something.